### PR TITLE
feat(search): Always search for page content, not only in app realm

### PR DESCRIPTION
**Pull request by mejo-**

### CODE REVIEW

File: lib/Search/PageContentProvider.php [hunk 1]
```diff
@@ -95,13 +95,7 @@ public function getOrder(string $route, array $routeParameters): int {
...
 	 */
 	public function search(IUser $user, ISearchQuery $query): SearchResult {
-		// Only search for page content if the app is active
-		if ($this->appManager->isEnabledForUser('circles', $user) &&
-			strpos($query->getRoute(), 'collectives.') === 0) {
-			$collectiveInfos = $this->collectiveHelper->getCollectivesForUser($user->getUID(), false, false);
-		} else {
-			$collectiveInfos = [];
-		}
+		$collectiveInfos = $this->collectiveHelper->getCollectivesForUser($user->getUID(), false, false);
```
Comment (juliushaertl) [inline lib/Search/PageContentProvider.php:98]: Do we still need the circles check or is that gracefully handled in getCollectivesForUser if circles is disabled?

**Review by juliushaertl — COMMENTED** [review]
Small question inline

We could actually also think about merging the two providers to only have one which displays matches on the page title first? But might be a bigger topic to work properly with pagination, so just consider this a future suggestion.

**Comment by mejo-** [comment]
> We could actually also think about merging the two providers to only have one which displays matches on the page title first? But might be a bigger topic to work properly with pagination, so just consider this a future suggestion.

Yeah, I'd say let's discuss this separately.

File: lib/Search/PageContentProvider.php (hunk 1 above)
Comment (mejo-) [reply lib/Search/PageContentProvider.php:98]: Nope, good catch. The check for circles is still required.

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**